backend/tests/integration/records/views/test_views_conferences.py

```diff
@@ -264,3 +266,63 @@ def test_date_range_contains_conferences_filter(api_client, db, es, create_recor
     assert 1 in found_recids
     assert 2 in found_recids
     assert 3 in found_recids
+
+
+@freeze_time(datetime(2019, 9, 15, 0, 0, 0))  # today is the 9th of September, 2019
+def test_conferences_start_date_range_filter(api_client, db, es, create_record):
```

Comment:
I'd separate the tests for each case, cleaner and easier to understand. wdyt?

Reply:
I was thinking about this but there is repetitive part for whole cases (creating 3-4 conferences) , which looked annoying little bit, but I can separate.

Probably pytest has better way to handle this, like test clases with `setup` method, but didn't wanna introduce it.